utils/constants.ts

```diff
@@ -15,3 +16,42 @@ export const PFP_WL_CONTRACTS_MAINNET = [
   "0x012f8e318fe04a1fe8bffe005ea4bbd19cb77a656b4f42682aab8a0ed20702f0",
   "0x01435498bf393da86b4733b9264a86b58a42b31f8d8b8ba309593e5c17847672",
 ];
+
+export enum NotificationType {
+  TRANSACTION = "TRANSACTION",
+}
+
+export enum TransactionType {
+  MINT_IDENTITY = "MINT_IDENTITY",
+  BUY_DOMAIN = "BUY_DOMAIN",
+  RENEW_DOMAIN = "RENEW_DOMAIN",
+  ENABLE_AUTORENEW = "ENABLE_AUTORENEW",
+  DISABLE_AUTORENEW = "DISABLE_AUTORENEW",
+  CHANGE_ADDRESS = "CHANGE_ADDRESS",
+  TRANSFER_IDENTITY = "TRANSFER_IDENTITY",
+  VERIFIER = "VERIFIER",
```

Comment:
Could we do a `VERIFIER` for POP and each social media. 

How I see it here is that users won't be able to see the difference between a Twitter `VERIFY` and a POP `VERIFY` right? Or maybe I don't understand something because in the subtext you say `discord verification` ...

Reply:
In starknet.quest we had this architecture because for each notification type we had a title "NFT received", and then a subtext that was customized with the name of the quest. 

I think here it could be nice to have a title that is specific to the type of notification, and then the subtext where we have specific information about the notification like if we renew a domain, having the domain name and the new expiry date, so something like : 
```
Domain renewed 
iris.stark until 2 Nov. 2024
```